default.nix

```diff
@@ -1,16 +1,32 @@
-with (import (fetchTarball https://github.com/NixOS/nixpkgs/archive/ffea68a09cedab941d19f02c9041689ebc81192e.tar.gz) {});
-# NOTE: when bumping nixpkgs, also update nixpkgs-src.json and .travis.yml
+let
+  # NOTE: when bumping nixpkgs, also update nixpkgs-src.json and .travis.yml
+  pkgs = import (fetchTarball https://github.com/nixos/nixpkgs/archive/fda4b93cd4fd3775408117c380aab0f33737d30f.tar.gz) {};
+  nodejs = pkgs.nodejs-8_x;
+in
 
-stdenv.mkDerivation {
-  name = "daedalus";
-
-  buildInputs = [
-    nix bash binutils coreutils curl
-    git python27 curl electron nodejs-8_x
-    nodePackages.node-gyp nodePackages.node-pre-gyp
-    gnumake
-  ];
+with (import (fetchTarball https://github.com/moretea/yarn2nix/archive/v1.0.0.tar.gz) { inherit pkgs nodejs; });
```

Comment:
Will this be in nixpkgs eventually?

Reply:
It is on nixpkgs master (https://github.com/NixOS/nixpkgs/pull/35340)